app/src/hooks/useCpk.tsx

```diff
@@ -1,4 +1,4 @@
-import CPK from 'contract-proxy-kit'
+import CPK from 'contract-proxy-kit/lib/esm'
```

Comment:
Changing this to explicitly use the ESM path because Webpack and Jest resolves 'contract-proxy-kit' differently.